website/project/model.py

```diff
@@ -2173,6 +2186,7 @@ def fork_node(self, auth, title='Fork of '):
         # Clear permissions before adding users
         forked.permissions = {}
         forked.visible_contributor_ids = []
+        forked.save()
```

Comment:
It needs to have been saved to create the `NotificationSubscription` before the forking user is added as a contributor. If not, [this](https://github.com/CenterForOpenScience/osf.io/pull/5033/files#diff-177f4640989f03049bb6b0eaedf8a53dR35) will be `None` and will throw an `AttributeError` on the next line.